### ISSUE TITLE
Info on stochasticity in Readme

### ISSUE THREAD

**Issue by ogencoglu** [pemistahl/lingua-py#105]
Is the library deterministic? This would be a useful info to add to Readme.
Cheers!

**Comment by pemistahl** [comment]
Yes, it is deterministic. For the same text, the same language is always returned.
